### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by basafilm**

### CODE REVIEW

File: week2/materials/homework/assignments.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+**Assignment 8:**
+
+The file `week2/assignments/products.json` needs to be uploaded Inside the bucket `hyf-serverless-course-week-2`. Upload the file through the CLI and write down the command needed.
+
+docs: `https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html`
+Commands: ""
+**Assignment 9:**
+Write down a brief use case on when S3 could be used for a data engineering assignment:
+
+use case: `?`
```
Comment (pdsorensen) [inline week2/materials/homework/assignments.md:40]: Please write a use case. 👍

File: week2/materials/homework/assignments.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+command: `aws s3api create-bucket --bucket hyf-serverless-course-week-2 --region us-east-1" , "aws s3 sync ./build s3://hyf-serverless-course-week-2`
+
+## Optional assignments:
+
+**Assignment 10:**
+
+What can be done to reduce the pricing for S3 when hosting a large number of files?
+
+brief description: `Clean up incomplete multipart uploads.`
```
Comment (pdsorensen) [inline week2/materials/homework/assignments.md:50]: Great overview!

File: week2/materials/homework/assignments.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+`Track the requests made to your bucket.`
+`Check for changes in the size of your bucket.`
+`Review the cost of individual buckets.`
+`Understand how your usage relates to your charges.`
+
+**Assignment 11:**
+
+There are many security features built into S3. Find your favourite feature, documentation for it, and explain briefly why.
+
+brief description of favourite security feature: `enable Server-side encryption settings`
```
Comment (pdsorensen) [inline week2/materials/homework/assignments.md:63]: Indeed, a very useful feature! Default encryption/safety with a single click.

**Review by pdsorensen — CHANGES_REQUESTED** [review]
Hi @basafilm , the PR is hard to review as the whole template is part of it. I suggest you finish the two assignments I commented, and then we can take care of the front end together in class. 

Cheers, great work!

**Review by bytewiz — APPROVED** [review]
Looks good to me 👍

File: week2/materials/homework/assignments.md [hunk 4]
```diff
@@ -24,9 +24,9 @@ Total size of website: `2.3 KB`
 
 Write down the main cost factors for S3
 
-brief description: `First 50 TB / Month	$0.023 per GB`
-`Next 450 TB / Month	$0.022 per GB`
-`Over 500 TB / Month	$0.021 per GB `
+brief description: `First 50 TB / Month $0.023 per GB`
+`Next 450 TB / Month $0.022 per GB`
+`Over 500 TB / Month $0.021 per GB `
```
Comment (bytewiz) [inline week2/materials/homework/assignments.md:30]: Would be nice to add the actual total price of 51TB here based on the the pricing you have above

51TB = ?

**Review by bytewiz — APPROVED** [review]
Looks good to me 👍

**Pull request merged**